sonar-plugin-api/src/main/java/org/sonar/api/resources/Project.java

```diff
@@ -27,14 +27,15 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang.builder.ToStringBuilder;
 import org.sonar.api.CoreProperties;
+import org.sonar.api.batch.fs.InputModule;
 import org.sonar.api.component.Component;
 import org.sonar.api.config.Settings;
 
 /**
- * A class that manipulates Projects in the Sonar way.
- *
  * @since 1.10
+ * @deprecated since 5.6 replaced by {@link InputModule}.
  */
+@Deprecated
 public class Project extends Resource implements Component {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)